pep/app/pods/components/modal-dialogs/help-tooltip/template.hbs

```diff
@@ -3,15 +3,15 @@
     @size="sm"
     @overlayClassNames="slide-card-overlay"
     @modal-class="slide-card-modal"
-    as |Modal|
+    @closable={{false}} as |Modal|
```

Comment:
fyi this modal is only used in mobile and it has a very different layout (its a "slide up card" type UI), so we might want to keep closable=true here for its improved UX, unless they explicitly asked to remove close-on-outside-tap behavior here too.

Reply:
They stated do it globally, so Im going to leave this for now and if they want it changed back we can easily do that